packages/lib/src/vis/rgb/RgbVis.tsx

```diff
@@ -34,19 +34,18 @@ function RgbVis(props: Props) {
   } = props;
 
   const values = useMemo(() => {
-    const typedDataArray = toTypedNdArray(dataArray, Uint8Array); // if `number[]`, assume uint8: [0, 255]
+    const typedDataArray = toTextureSafeNdArray(dataArray);
```

Comment:
This introduces a regression: it removes support for `uint8` arrays. See the mandrill image in https://h5web.panosc.eu/?file=grove.h5 that is now blank.

Reply:
Looks like supporting `uint8` normalized texture is useful here.

Reply:
Indeed, same problem with the grey that was cropped by mistake in the mock datasets.

Reply:
At least `mandrill_float` works now :joy:

Reply:
I've updated the PR description.